spec/presenters/two_factor_authentication/set_up_piv_cac_selection_presenter_spec.rb

```diff
@@ -42,7 +42,7 @@
     end
 
     context 'with a confirmed email address ending in .gov or .mil' do
-      let(:user) { create(:user, email: 'example@example.gov') }
+      let(:user) { create(:user, email: 'example@gsa.gov') }
```

Comment:
Will this test continue to work when we remove the feature flag? I think this is a good case for setting the default configuration value to `true`, and overriding it to `false` in production. I suspect we'll need some way to seed default federal domains for the test environment.

Reply:
Could we set the Test to true instead, and seed the federal domains?

Reply:
Yeah what I was thinking was:

```yaml
# application.yml.default

use_fed_domain_class: true

production:
  use_fed_domain_class: false
```

which would effectively set it to `true` in the tests, and likely cause this to start failing.

Reply:
We can also figure that out later, I'm not too concerned about it. That can be a clean-up task for removing the feature flag. I dunno that it's necessarily better or worse for this to be enabled "by default" in the tests.